tests/testthat/test-get_code.R

```diff
@@ -697,6 +697,32 @@ testthat::test_that("data() call is returned when data name is provided as a cha
 })
 
 testthat::describe("Backticked symbol", {
+  testthat::it("code can be retrieved with get_code", {
+    td <- teal_data() |>
```

Comment:
No pipe operator before upgrading to R 4.1

Reply:
Great catch, sorry about this